### ISSUE TITLE
Exchange rate not updating when using Tor

### ISSUE THREAD

**Issue by dmp1ce** [mycelium-com/wallet-android#355]
I'm using Orbot as my connection in Mycelium and I cannot get an updated price unless I switch to using https instead of Tor.

I have tried several exchanges including BitcoinAverage, Kraken and Coinbase but all I see for the price is "Exchange rate not available for BitcoinAverage" and then I get a "No server connection" popup error.

I expect the BitcoinAverage price to be updated without errors after a few seconds when using Tor.

**Comment by Giszmo** [comment]
Can confirm. Somtething is weird. Spinners forever here. Will look into it.

**Comment by Giszmo** [comment]
@DanielWeigl not sure what I did here or was supposed to do but I rebooted node2 after not getting appropriate responses to a faulty curl query. Then I still had to `systemctl restart nginx.service tor.service` to make it work with `curl -k -X POST -H "Content-Type: application/json" -d '{}' --proxy socks5h://127.0.0.1:9050 https://slacef5ylu6op7zc.onion/wapi/wapi/getMinerFeeEstimations | jq`.

For me it works again but as our other endpoint works with the correct script without me touching anything, I wonder if I did anything relevant at all.

As zabbix missed the issue, I wonder how to extend it to cover this issue. Also I did not find anything related in the wapi logs.
